<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="4" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>252</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_34">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Comment</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="2" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="result_meta_form_layout">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="7" rowspan="2">
        <widget class="QPushButton" name="result_sample_edit_button">
         <property name="text">
          <string>edit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6" rowspan="2">
        <widget class="QPushButton" name="result_sample_delete_button">
         <property name="text">
          <string>Delete Sample</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QComboBox" name="id_combo">
         <property name="minimumSize">
          <size>
           <width>194</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>194</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="4" rowspan="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>252</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="4" colspan="5">
        <widget class="QWebEngineView" name="result_plot_view" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="result_meta_update_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="7" column="8">
        <widget class="QPushButton" name="prop_download">
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="8">
        <widget class="QTextEdit" name="sample_meta_24">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="9">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Project</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sample_meta_date">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Date</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sample_meta_experimenter">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Person</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sample_meta_project">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="prop_combo">
         <property name="minimumSize">
          <size>
           <width>194</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>194</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="3" rowspan="2">
        <widget class="QListView" name="result_file_list_view">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="result_meta_edit_button">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>edit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="8" rowspan="2">
        <widget class="QPushButton" name="result_sample_update_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="7" column="6">
        <widget class="QPushButton" name="result_meta_delete_button">
         <property name="text">
          <string>Delete Data</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="9">
        <layout class="QGridLayout" name="sample_meta_grid">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>result_meta_update_button</tabstop>
  <tabstop>result_meta_edit_button</tabstop>
  <tabstop>prop_combo</tabstop>
  <tabstop>result_sample_update_button</tabstop>
  <tabstop>prop_download</tabstop>
  <tabstop>id_combo</tabstop>
  <tabstop>result_sample_edit_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>prop_combo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>mode_meta_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>id_combo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>id_meta_change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prop_download</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>data_download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>805</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>result_sample_edit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result_sample_edit_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>result_sample_update_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result_sample_update_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>result_meta_edit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result_meta_edit_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>805</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>820</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>result_meta_update_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result_meta_update_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>805</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>820</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>result_meta_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result_meta_delete_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>805</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>820</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>result_sample_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result_sample_delete_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>664</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>result_file_list_view</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>result_file_list_view_double_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>820</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>data_download()</slot>
  <slot>mode_meta_change()</slot>
  <slot>id_meta_change()</slot>
  <slot>result_sample_edit_button_clicked()</slot>
  <slot>result_sample_update_button_clicked()</slot>
  <slot>result_meta_edit_button_clicked()</slot>
  <slot>result_meta_update_button_clicked()</slot>
  <slot>result_meta_delete_button_clicked()</slot>
  <slot>result_sample_delete_button_clicked()</slot>
  <slot>result_meta_rep_show_points_button_clicked()</slot>
  <slot>result_meta_data_show_button_clicked()</slot>
  <slot>result_file_list_view_double_clicked()</slot>
 </slots>
</ui>
